keras/applications/applications_test.py

```diff
@@ -235,5 +242,11 @@ def _get_output_shape(model_fn):
     return model.output_shape
 
 
+def _get_last_layer_activation(model_fn):
```

Comment:
This doesn't necessary to be a separate function, just make it part of test_application_classifier_activation